### PR TITLE
Collected small changes and fixes

### PR DESCRIPTION
**Pull request by akohlmey**
**Summary**

This pull request combines multiple small changes and bug fixes that do not warrant individual pull requests

**Related Issue(s)**

N/A

**Author(s)**

Axel Kohlmeyer, Temple U

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

N/A

**Implementation Notes**

The following individual changes are included:
- avoid segfault in multi-threaded neighbor lists, if the number of threads has been reduced during the executable run (e.g. when using LAMMPS GUI)
- small doc updates for fix srd, pair style reaxff, and pair style tmd/ilp
- port of LAMMPS GUI to Qt6
- replace entire word under cursor with auto-completion
- avoid infinite loop in StdCapture and instead time out after one second
- avoid multiple out of bounds memory accesses in LAMMPS GUI detected by compiling to Qt with assertions enabled
- auto detect OpenMP availability when compiling LAMMPS GUI as standalone executable

**Post Submission Checklist**

- [x] The feature or features in this pull request is complete
- [x] Licensing information is complete
- [x] Corresponding author information is complete
- [x] The source code follows the LAMMPS formatting guidelines
- [ ] Suitable new documentation files and/or updates to the existing docs are included
- [ ] The added/updated documentation is integrated and tested with the documentation build system
- [ ] The feature has been verified to work with the conventional build system
- [ ] The feature has been verified to work with the CMake based build system
- [ ] Suitable tests have been added to the unittest tree.
- [ ] A package specific README file has been included or updated
- [ ] One or more example input decks are included

### CODE REVIEW

File: lib/pace/Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 SHELL = /bin/sh
+CMAKE= cmake
```
Comment (stanmoore1) [inline lib/pace/Makefile]: @akohlmey can you wait on this change? I submitted a PR that adds the makefile back in here: https://github.com/ICAMS/lammps-user-pace/pull/19 which doesn't require CMake.

**Review by stanmoore1 — CHANGES_REQUESTED** [review]

File: lib/pace/Makefile (hunk 1 above)
Comment (akohlmey) [reply lib/pace/Makefile]: Sure. You need the access permissions changed, too, right? So we would need another PR from Yuri before we can finish the release anyway.

Comment (stanmoore1) [reply lib/pace/Makefile]: Right

**Review by oywg11 — APPROVED** [review]

File: lib/pace/Makefile (hunk 1 above)
Comment (akohlmey) [reply lib/pace/Makefile]: The lib/pace/Makefile changes have been reverted.

Comment (stanmoore1) [reply lib/pace/Makefile]: Thanks!

**Review by stanmoore1 — APPROVED** [review]
I approve

**Pull request merged**